### ISSUE TITLE
Broken intall for python 3.11

### ISSUE THREAD

**Issue by alexaquila** [drufat/triangle#73]
Current install via pip seems to be broken for python 3.11:
```  
   triangle/core.c:196:12: schwerwiegender Fehler: longintrepr.h: Datei oder Verzeichnis nicht gefunden
        196 |   #include "longintrepr.h"
            |            ^~~~~~~~~~~~~~~
```
Seems to be a breaking change in cpython
https://github.com/cython/cython/pull/4428
https://github.com/python/cpython/pull/28968
Apparently, including Python.h should be enough, now:
https://github.com/python/cpython/blob/34e93d3998bab8acd651c50724eb1977f4860a08/Include/Python.h#L53

**Comment by SorooshMani-NOAA** [comment]
@drufat may I ask that you please release a new version for `triangle`? The fix for the `Python 3.11` issue is already merged, but it the package is not released after the merge.

**Comment by drufat** [comment]
New version released.
